lib/fuse_i.h

```diff
@@ -9,6 +9,13 @@
 #include "fuse.h"
 #include "fuse_lowlevel.h"
 
+/**
+ * Fsync flags
+ *
+ * FUSE_FSYNC_FDATASYNC: Sync data only, not metadata
+ */
+#define FUSE_FSYNC_FDATASYNC	(1 << 0)
```

Comment:
Is this change intentional? It doesn't seem to belong to the feature being added.